src/cpp/rediscluster.cpp

```diff
@@ -327,10 +327,12 @@ CommandReply RedisCluster::set_model(const std::string& model_name,
     for ( ; node != _db_nodes.cend(); node++) {
         // Build the node prefix
         std::string prefixed_key = "{" + node->prefix + "}." + model_name;
+        std::vector<std::string> tmp_inputs = _get_tmp_names(inputs, node->prefix);
```

Comment:
Is there are a reason why we are doing _get_tmp_names on these?

Reply:
This is a bug; fixed